liquibase-standard/src/test/resources/liquibase/test-changelog-text-clob-types.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
This changelog is not being used anywhere. Is your intention to get this executed as part of our integration tests? If yes, I think we should move it to that other module and add a new integration test to make use of it. Another option here is to add the changesets of this file to `common.tests.changelog.xml`, which contains general scenarios that will be executed against all the DBs we run integration tests for.